runtime/src/lib.rs

```diff
@@ -201,20 +252,16 @@ impl pallet_balances::Trait for Runtime {
 }
 
 parameter_types! {
-    pub const TransactionBaseFee: Balance = 100 * MICRO_RAD;
     pub const TransactionByteFee: Balance = 1 * MICRO_RAD;
-	// setting this to zero will disable the weight fee.
-	pub const WeightFeeCoefficient: Balance = 100_000_000;
 	// for a sane configuration, this should always be less than `AvailableBlockRatio`.
-	pub const TargetBlockFullness: Perbill = Perbill::from_percent(25);
+	pub const TargetBlockFullness: Perquintill = Perquintill::from_percent(25);
 }
 
 impl pallet_transaction_payment::Trait for Runtime {
 	type Currency = Balances;
 	type OnTransactionPayment = Author;
-	type TransactionBaseFee = TransactionBaseFee;
 	type TransactionByteFee = TransactionByteFee;
-	type WeightToFee = LinearWeightToFee<WeightFeeCoefficient>;
+	type WeightToFee = IdentityFee<Balance>;
```

Comment:
It is the same implementation with some code refactor which was moved to `frame_support`

Reply:
I just looked into substrate, and it seems that with dropping the `WeightFeeCoefficient`, the weights in substrate were increased as well. 

Two examples: 

Module | Call | Weight | Weight New (RC3)
-- | -- | -- | -- 
balances | transfer | 1_000_000 | T::DbWeight::get().reads_writes(1, 1) + 70_000_000 = 25_000_000 + 100_000_000 + 70_000_000 = 195_000_000
democracy | propose | 5_000_000 | 50_000_000 + T::DbWeight::get().reads_writes(2, 3) = 50_000_000 + 50_000_000 + 300_000_000 = 400_000_000

That implies that we should adjust our fees accordingly. What confuses me a bit is the fact that Substrate's WeightFeeCoefficient was previously `1_000` (ours was `100_000_000`), but the new weights are only increased by a factor of 100 to 200.

On top of that, since we use different decimals, we would still need to implement a WeightFeeCoefficient.

@vedhavyas do you know whether any of the other testnets of Substrate/Polkadot have 18 decimals as we do?

Reply:
@philipstanislaus could you point me where `T::DbWeight::get().reads_writes(1, 1)` gets added as part of the weight ?